src/components/Form/index.js

```diff
@@ -1,12 +1,14 @@
+import Button from '../Button';
+
 import StyledForm from './styled';
 
 export default function Form({onSubmit, placeholderText, submitButtonText}) {
 	return (
 		<StyledForm onSubmit={onSubmit}>
 			<input type="text" name="inputField" placeholder={placeholderText} required />
-			<button type="submit" value="Submit the form">
+			<Button submitButton={true} type="submit" value="Submit the form">
```

Comment:
Why do you double the submit information? If you declare your button as type submit you don't need to set an attribute called submit button as true.

Reply:
Thanks for the Information! :)